captum/attr/_core/deep_lift.py

```diff
@@ -310,10 +356,10 @@ def forward_fn():
             forward_fn.device_ids = forward_func.device_ids
         return forward_fn
 
-    def _is_non_linear(self, module):
+    def _is_non_linear(self, module: nn.Module) -> bool:
         return type(module) in SUPPORTED_NON_LINEAR.keys()
 
-    def _forward_pre_hook_ref(self, module, inputs):
+    def _forward_pre_hook_ref(self, module: Any, inputs: TensorOrTupleOfTensors) -> None:
```

Comment:
This module should also be Module, note that this may require another "# type: ignore", since we add an attribute to the Module, which is not typed in the Module.